### DISCUSSION TITLE
Welcome to the AstroImageJ User Forum on GitHub discussions!

### DISCUSSION THREAD

**Discussion by keastrid** [AstroImageJ/astroimagej#61] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by michealroberts** [comment]
Hello 👋 Long time user of AstroImageJ, thank you very much for providing this tool to the community. 🔭 

I'm currently using AstroImageJ and ASTAP as a benchmarking tool for a low-level Go lang API for plate solving (taking approximately <2/10ths of a second to solve), [SkySolve](https://github.com/observerly/skysolve) that I will be open sourcing imminently.

**Reply by karenacollins** [reply]
Keep us posted when you go open source.

**Reply by michealroberts** [reply]
Hey @karenacollins I've started to open source the tooling we have created here: https://github.com/observerly/skysolve

I'm just working on inverse SIP parameters, before moving onto supporting splines, and some other API cleanup. We'd like to make it truly zero-dependency by remove all observerly related Go packages.
